authorization-server/src/modules/auth/auth.controller.ts

```diff
@@ -164,8 +164,9 @@ export class AuthController {
   async logout(
     @Body() body: LogoutDto,
     @Res({ passthrough: true }) res: Response,
+    @Req() req: Request,
   ): Promise<void> {
-    const tokenDecoded = decodeJWT(body.refreshToken) as IRefreshTokenPayload;
+    const tokenDecoded = decodeJWT(req.user as string) as IRefreshTokenPayload;
```

Comment:
This approach works because `valid-refesh-token.guard` is adding the refresh token to the request, right?

Reply:
@jrhender Correct!